Source/Core/Runtime/Configuration/RuntimeConfigurator.cs

```diff
@@ -133,47 +133,50 @@ public static BaseRuntimeConfiguration Configuration
         {
             get
             {
-                if (Instance.runtimeConfiguration != null)
+                if (Instance?.runtimeConfiguration != null)
                 {
                     return Instance.runtimeConfiguration;
                 }
 
-                Type type = ReflectionUtils.GetTypeFromAssemblyQualifiedName(Instance.runtimeConfigurationName);
+                Type type = ReflectionUtils.GetTypeFromAssemblyQualifiedName(Instance?.runtimeConfigurationName);
 
                 if (type == null)
                 {
-                    Debug.LogErrorFormat("IRuntimeConfiguration type '{0}' cannot be found. Using '{1}' instead.", Instance.runtimeConfigurationName, typeof(DefaultRuntimeConfiguration).AssemblyQualifiedName);
+                    Debug.LogWarning($"IRuntimeConfiguration type '{Instance?.runtimeConfigurationName}' cannot be found. Using '{typeof(DefaultRuntimeConfiguration).AssemblyQualifiedName}' instead.");
                     type = typeof(DefaultRuntimeConfiguration);
                 }
 
                 Configuration = (BaseRuntimeConfiguration)ReflectionUtils.CreateInstanceOfType(type);
 
-                return Instance.runtimeConfiguration;
+                return Instance?.runtimeConfiguration;
             }
             set
             {
                 if (value == null)
                 {
-                    Debug.LogError("Process runtime configuration cannot be null.");
+                    Debug.LogWarning("Process runtime configuration cannot be null.");
```

Comment:
Actually I wouldn't want to change the "Runtime configuration missing" errors to warnings. I would argue they are legitimate errors.
I'm ok with the warning that "there is no registry so the object will not be registered", but is there a way to avoid this? It might be possible to check RuntimeConfigurator.Exists in the Process Scene Object instead of calling the getter/setter and getting here.